### PR TITLE
Manual version and change log update after release

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Version update for packages:

azure-identity
azure-storage-blob
azure-storage-file-datalake
azure-storage-file-share
azure-keyvault-keys
azure-keyvault-secrets
azure-keyvault-certificates

### CODE REVIEW

File: sdk/identity/azure-identity/setup.py [hunk 1]
```diff
@@ -51,7 +51,7 @@
     author_email="azpysdkhelp@microsoft.com",
     url="https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity",
     classifiers=[
-        "Development Status :: 5 - Production/Stable",
+        'Development Status :: 5 - Production/Stable',
```
Comment (chlowell) [inline sdk/identity/azure-identity/setup.py]: Unintended side effect of script or regex?

Comment (praveenkuttappan) [reply sdk/identity/azure-identity/setup.py]: Yes and No. Auto-versioning tool updates classifier if it doesn't match and keep everything with single quote. Do you want this to be updated to old one? It's anyway going to be updated when version bump is triggered automatically.

Comment (chlowell) [reply sdk/identity/azure-identity/setup.py]: We format with black, which will change this back to double quotes and allows no configuration of that behavior. I suggest our auto-versioning tool should use double quotes as well.

Comment (praveenkuttappan) [reply sdk/identity/azure-identity/setup.py]: @chlowell : I have fixed versioning tool as well as setup.py files as per your suggestion.

**Comment by chlowell** [comment]
Looks good to me but you have a merge conflict. Whatever you do to fix that will dismiss my approving review.

**Comment by praveenkuttappan** [comment]
@chlowell , @annatisch, @xiafu-msft : Can you please have a look at this PR for version update and approve?

**Pull request merged**